.env.example

```
@@ -0,0 +1,65 @@
+# Shared configuration for host backend commands and Docker Compose.
+# Copy this file to .env at the repository root and replace production secrets.
+# Use single quotes for literal values containing spaces, #, or $. Do not use
+# ${VAR} expansion: host and Compose processes must receive the same literal.
+
+# PostgreSQL. The backend safely builds DATABASE_URL from these components when
+# DATABASE_URL is empty. Compose overrides only host/port for container access.
+POSTGRES_DB=nexaflow
+POSTGRES_USER=nexaflow
+POSTGRES_PASSWORD=nexaflow
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)